client/app/following/following.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+angular.module('followingList', ['ngRoute'])
+  .component('followingList', {
+    templateUrl: 'following/following.html',
+    controller: ['$http', '$rootScope', function TweetListController($http, $rootScope) {
+        var self = this;
+
+        
+
+      const requestOptions = {
+          headers: { 'X-session': $rootScope.x_session }
+      };
+
+      $http.get('http://localhost:8080/twitterapi/following/', requestOptions).then(function (response) {
+          self.followings = response.data;
+          
+      });
+        //example base on  home.js
+        //give me some point
+        //
+      $http.post('http://localhost:8080/twitterapi/follow/', requestOptions).then(function (response) {
```

Comment:
you should pass the name of the user that you want to follow/unfollow in to requestOptions.